hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -217,6 +217,36 @@ static const struct usage_map xpadneo_usage_maps[] = {
 
 	/* disable duplicate button */
 	USAGE_IGN(0xC0224),
+
+	/* XBE2 hacks */
+	/* Disable "dial", which is a redundant representation of the D-Pad */
+	USAGE_IGN(0x10037),
+
+	/* Disable unnecessary axes Vx, Vy, Vz, Vbrx, Vbry, Vbrz which cause an additional axis to show up */
+	USAGE_IGN(0x10040),
+	USAGE_IGN(0x10041),
+	USAGE_IGN(0x10042),
+	USAGE_IGN(0x10043),
+	USAGE_IGN(0x10044),
+	USAGE_IGN(0x10045),
+
+	/* Button 319 (copy of A button) */
+	USAGE_IGN(0x90010),
+	
+	/* TRIGGER_HAPPY1 (copy of B button) */
```

Comment:
It doesn't make much sense to put the original symbol names here as these don't have a real meaning when reading the events - we are remapping stuff anyways. So it's sufficient to say "remove duplicate button A" in the final patch.

Any maybe also make this appear as one semantic block, put a header comment "remove duplicate buttons" and put one comment inline with each line "duplicate of A" etc (similar to the beginning of the array).